graphrag/index/update/files.py

```diff
@@ -0,0 +1,50 @@
+# Copyright (c) 2024 Microsoft Corporation.
+# Licensed under the MIT License
+
+"""File management and utils for Incremental Indexing."""
+
+from typing import NamedTuple
+
+import pandas as pd
+
+from graphrag.index.storage.typing import PipelineStorage
+from graphrag.utils.storage import _load_table_from_storage
+
+
+# Output delta named tuple
+class InputDelta(NamedTuple):
+    """Named tuple for output delta."""
+
+    new_inputs: pd.DataFrame
+    deleted_inputs: pd.DataFrame
+
+
+async def get_delta_docs(
```

Comment:
does this needs to be async?

Reply:
Oh i see you need to use await on _load_table_from_storage

Reply:
Yup :( All storage related operations are async on the implementations